imi/src/main/java/org/motechproject/nms/imi/web/ImiController.java

```diff
@@ -103,6 +102,13 @@ private static boolean validateCdrFileInfo(StringBuilder errors, FileInfo fileIn
     }
 
 
+    private void verifyFileExistsInAuditRecord(String fileName) {
+        List<FileAuditRecord> records =  fileAuditRecordDataService.findByFileName(fileName);
```

Comment:
If you are just looking for the existence of a record use the auto generate count() method.